src/logic/shorten-url.js

```diff
@@ -39,9 +39,12 @@ class BitlyResponseError extends Error {
 }
 
 export async function shortenUrl(longUrl: string): Promise<string> {
-  const log = getLogger('shortenUrl');
+  const log = getLogger('logic.shorten_url.shorten_url');
```

Comment:
This seems more reasonable `slug.slug_slug.slug` approach seems more reasonable given the restrictions.

Reply:
I wasn't so happy for this specific case; maybe the file name should be changed to better reflect what's in it... bitly.js comes to my mind. #nitpicking :-)